### ISSUE TITLE
Making first link visible. 

### ISSUE THREAD

**Issue by dmoskovtsov** [fisshy/react-scroll#128]
Hi, thanks for the great library!

I noticed that Test1 button in then demo: ( http://fisshy.github.io/react-scroll-example/basic/index.html ) is not selected by default which is part of requirement I have for my page. What would be the way I make it selected immediately after page is loaded?

**Comment by fisshy** [comment]
Hi, thanks!

The demopage is not updated to the latest version

In you page you can call "scrollSpy.update();"

```
var React   = require('react');
var Scroll  = require('react-scroll');
var scrollSpy  = Scroll.scrollSpy;

var Page= React.createClass({
  componentDidMount: function() {
    scrollSpy.update();
  },
});
```

**Comment by dmoskovsov** [comment]
Thanks for the prompt replay! This helped. I'm having one more issue. 

I need to have 2 links pointed to 1 element. And of of links (Signup) only when it's not under "signup" element. You can see more details at screenshots: 

https://www.evernote.com/l/AEfREglY1ktNCoWaSG9CDKbylK_8V642eqU

https://www.evernote.com/l/AEfLa7bzyMpDz4e3GGBbR7ADwHdfV45ObBA

Looks like you can have only 1 link per Element. How do I solve it? 

My code: 

```    
AnchorNavigation

 links = [{href: 'signup', messageId: 'ShopBack Rewards'},
        {href: 'education', messageId: 'How to Earn Cashback'},
        {href: 'widget', messageId: 'Exclusive Promotions'}];

renderLinks = (links) => {
        return links.map((linkData, idx) => {
            return (
                <li key={'anchor-href-' + idx}>
                    <Link offset={idx === 0 ? -50 : 0 }
                          to={linkData.href}
                          spy={true}
                          smooth={true}>
                        {linkData.messageId}
                    </Link>
                </li>
            );
        });
    };
```

```
Partner

            <Row>

                <AnchorNavigation links={this.links}/>

                <Signup/>

                <Education/>
               
            </Row>

```

```
Partner, Education 
            <Element 
                     name="signup">
                
            </Element>

```

**Comment by fisshy** [comment]
With "DirectLink" you can skip the "name" on the elements and use "id" only and then you can map as many as you want.

**Comment by dmoskovsov** [comment]
I changed my code to DirectLink and element to use id and it didn't help. Exactly the same behaivior. I also tried to run examples. But after I do "npm run examples" I see no output. You can see it here: 

https://www.evernote.com/l/AEdZZki1EfxClYlsWPMk-bOe-cASMtoPLoE

**Comment by dmoskovsov** [comment]
Hi, any help with this one?

**Comment by fisshy** [comment]
Hi, sorry for the delay.

I solved it with this 

```
<Link activeClass="active" className="test8" to="same" spy={true} smooth={true} duration={500}>Same target</Link>
<Link activeClass="active" className="test9" to="same" spy={true} smooth={true} duration={500}>Same target</Link>

<Element id="same" className="element">
  Two links point to this
</Element>

```

**Comment by raviteja83** [comment]
scrollSpy.update(); does not set the first link active sometimes.how to fix that

**Comment by FrailWords** [comment]
I am facing the same issue that `scrollSpy.update()` is not selecting the first link by default.  Is there an update or workaround for this issue?

**Comment by fisshy** [comment]
This one should be solved in v1.6.7

**Comment by FrailWords** [comment]
@fisshy For some reason, the upgrade still didn't do it for me.  What worked for me was basically storing the `ref` value of the first `Link` element and using `Scroll.scroller.setActiveLink()` function to make it active on component mount.
